app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,79 @@
+
+.page-logo {
+  text-align: center;
+}
+
+.img {
+  // margin-top: -10px;
+  // display: block;
+  width: 250px;
+  height: 250px;
+  background-color: #cccccc;
+  background: url("http://s9.postimg.org/h1lnkqwzz/image.jpg");
```

Comment:
URL `http://s9.postimg.org/h1lnkqwzz/image.jpg` should not contain protocol or domain